tests/components/glances/test_config_flow.py

```diff
@@ -54,6 +57,21 @@ async def test_form_cannot_connect(hass: HomeAssistant, mock_api: MagicMock) ->
     assert result["errors"] == {"base": "cannot_connect"}
 
 
+async def test_form_invalid_auth(hass: HomeAssistant, mock_api: MagicMock) -> None:
+    """Test to return error if not authroized."""
+
+    mock_api.return_value.get_ha_sensor_data.side_effect = GlancesApiAuthorizationError
+    result = await hass.config_entries.flow.async_init(
+        glances.DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input=MOCK_USER_INPUT
+    )
+
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] == {"base": "invalid_auth"}
```

Comment:
Also, we like to have all config flow tests onwards to finish in either CREATE_ENTRY or ABORT to show the forms are able to recover from an error. (so the trick is after this line is to make sure you input data again and make it succeed)